### ISSUE TITLE
Incorrect solution for fizz and buzz

### ISSUE THREAD

**Issue by KhalilKhunji** [Muhannedo/js-arrays-lab#2]
https://github.com/Muhannedo/js-arrays-lab/blob/fd64da0052370292aebffb7f606916797667b074/app.js#L45-L53

While your result for fizzbuzz is correct, fizz and buzz were not.

This is because you chained the if statements with else ifs; so that if during the loop a condition was fulfilled, it would only push the number into _one_ of the arrays, when in fact you should be pushing them into any number of arrays so long as the condition for being part of that array is fulfilled.

In other words, it should be something like:

```
for (num of numbers) {
  if (num % 15 === 0) {
    fizzbuzz.push(num);
  } 
  if (num % 5 === 0) {
    buzz.push(num);
  } 
  if (num % 3 === 0) {
    fizz.push(num);
  }
}
```

Can you see the difference there?

No other issues with your lab; good job!

P.S. If I could give one more point of advice - practice using forEach, as it's very useful as far as looping methods go in Javascript.
